styles/styles.css

```diff
@@ -0,0 +1,300 @@
+body {
+  margin: 10px;
+  padding: 5px;
+  display: grid;
+  grid-template-rows: repeat(8, 200px);
+  grid-template-columns: repeat(auto-fill, minmax(100%, 1fr));
+  grid-template-areas:
+  "Header "
+  "Intro"
+  "Quote"
+  "Why"
+  "Quote2"
+  "Team"
+  "Sponsors"
+  "Footer footer";
+  background-image: url("../images/2.jpg");
```

Comment:
Nice touch with all the images! I noticed how you picked images that fit the size requirements. This was more than what the project required. The way you linked the images to the CSS file was clever! Overall nice visual effect!